runtime-common/stdlib/crypto/crypto-functions.cpp

```diff
@@ -0,0 +1,19 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2025 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-common/stdlib/crypto/crypto-functions.h"
```

Comment:
Let's add include of `runtime-core.h`. Indirect includes can lead to compilation fails in case of some specific modifications in the header file

Reply:
fixed